backend/src/main/resources/application-aws.yml

```
@@ -50,4 +61,13 @@ weather:
-  key: ${AES_KEY}
\ No newline at end of file
+  key: ${AES_KEY}
+
+ffmpeg:
+  path: /usr/bin/ffmpeg
+ffprobe:
+  path: /usr/bin/ffprobe
+
+slack:
+  webhook:
+    url: https://hooks.slack.com/services/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
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)